templates/frontend/objects/galley_link.tpl

```diff
@@ -48,8 +48,21 @@
 	{/if}
 {/if}
 
+{* Revision? *}
+{if $isPreviousRevision}
+	{assign var=path value=$parentId|to_array:$submissionRevision:$galley->getBestGalleyId()}
+	{assign var=label value=$galley->getGalleyLabel()}
+{else}
+	{assign var=path value=$parentId|to_array:$galley->getBestGalleyId()}
+	{assign var=label value=$galley->getGalleyLabel()}
```

Comment:
the label is always the same i.e. two same lines of code. thus maybe not to use a variable for that?